plugins/qtosgrave/osgviewerwidget.cpp

```diff
@@ -32,6 +33,149 @@
 
 namespace qtosgrave {
 
+class OpenRAVETracker : public osgGA::NodeTrackerManipulator {
+public:
+    OpenRAVETracker()
+    {
+        _currentTransitionAnimationTime = 0;
+        _transitionAnimationPath = new osg::AnimationPath();
+        _transitionAnimationPath->setLoopMode(osg::AnimationPath::NO_LOOPING);
+    }
+
+    void TrackNode(osg::Node* node, const osg::Vec3d& offset, double trackDistance, osg::Camera* currentCamera, const osg::Vec3d& worldUpVector)
```

Comment:
@pablocael if there's no reason to use raw pointer, let's not. You can do `osg::Node& node`. thanks

Reply:
Used `OsgNodePtr` instead

Reply:
Rosen, we changed to ref pointer since we are using ref pointers as arguments in many other places already. Copying the ref pointer is ok since it will copy the ref count. Its safer than raw pointer (if any exception happens, for instance).